### ISSUE TITLE
Timed out after 5000ms

### ISSUE THREAD

**Issue by tshelianthus** [mapbox/mapbox-studio-classic#1366]
Timed out after 5000ms

**Comment by geotrek** [comment]
any resolution?

**Comment by wilhelmberg** [comment]
@tshelianthus @geotrek 
Are you behind a proxy?

**Comment by geotrek** [comment]
Yes, we tried the proxy enabled.exe and that didn't work. We had the regular .exe. 0.2.7 working, and then the timed out after 5000ms error just started showing up?

**Comment by wilhelmberg** [comment]
@geotrek 

> We had the regular .exe. 0.2.7 working, and then the timed out after 5000ms error just started showing up?

0.2.7 never worked with proxies.

Maybe something changed with your internet connection or this is due to slow network between you and tile.mapbox.com or some unexpected latency on the mapbox end.

Can you let us know if it still persists next time you try the app?

Otherwise you could try the new proxy enabled test version I just published.
Download links and instructions: https://github.com/mapbox/mapbox-studio/issues/860#issuecomment-112001682
